doc/reference/guidelines.rst

```diff
@@ -0,0 +1,48 @@
+.. highlight:: python
+
+=================
+Coding Guidelines
+=================
+
+Python
+======
+
+- PEP8 ignore E501
+- prefer % over format
+- try to avoid generators and decorators
+
+
+File naming
+===========
+
+Split the business logic by sets of models, in each sets select a main
+model, this model gives its name to the set. If there is only one set of module
+it's name is the same as the module name.
+
+For each set named <main_model> the following files may be created:
+
+data/<main_model>.xml
```

Comment:
What about data and demo ? I use data/<main_model>_data.xml and data/<main_model>_demO.xml .